### PR TITLE
Show Inscription Number

### PR DESCRIPTION
**Pull request by habibitcoin**
Example: 
image

https://github.com/dannydeezy/nosft/issues/30

### CODE REVIEW

File: src/components/ordinal-card/index.js [hunk 1]
```diff
@@ -48,7 +48,12 @@ const OrdinalCard = ({ overlay, price, type, utxo, authors, confirmed, date, onS
 
     return (
         <div className={clsx("product-style-one", !overlay && "no-overlay")}>
-            <div className="card-thumbnail">{renderImage()}</div>
+            <div className="card-thumbnail">
+                {renderImage()}
+                <div className="inscription-details-area">
+                    <div className="inscription-number">No. {utxo.inscriptionNumber}</div>
```
Comment (dannydeezy) [inline src/components/ordinal-card/index.js:54]:
```suggestion
                    <div className="inscription-number">#{utxo.inscriptionNumber}</div>
```

Comment (dannydeezy) [reply src/components/ordinal-card/index.js:54]: suggesting using `#` instead of `No.`

**Comment by dannydeezy** [comment]
looks good, one suggestion + merge conflics

File: src/components/ordinal-card/index.js (hunk 1 above)
Comment (dannydeezy) [reply src/components/ordinal-card/index.js:54]: also wondering if you can get the padding on the left to lineup with the rest of the items below it in the card

**Comment by Uthpala** [comment]
Nice work 💯

**Review by andredevjs — APPROVED** [review]

**Pull request merged**